authority-portal-frontend/src/app/pages/control-center-organization-edit-page/state/control-center-organization-edit-page-state-impl.ts

```diff
@@ -43,24 +44,29 @@ export class ControlCenterOrganizationEditPageStateImpl {
     private apiService: ApiService,
     private formBuilder: FormBuilder,
     private customRxjsOperators: CustomRxjsOperators,
+    private globalStateUtils: GlobalStateUtils,
   ) {}
 
   @Action(Reset, {cancelUncompleted: true})
   onReset(ctx: Ctx, action: Reset): Observable<never> {
-    return this.apiService.getOwnOrganizationDetails().pipe(
-      Fetched.wrap({failureMessage: 'Failed to fetch user details'}),
-      tap((organization) => {
-        ctx.patchState({
-          organization,
-          headerBarConfig: organization
-            .map((data) => this.buildHeaderBarConfig(data))
-            .orElse(null),
-        });
-        action.setFormInComponent(
-          organization.map((data) => this.rebuildForm(data)).orElse(null),
-        );
-      }),
-      ignoreElements(),
+    return this.globalStateUtils.getDeploymentEnvironmentId().pipe(
+      switchMap((environmentId) =>
+        this.apiService.getOwnOrganizationDetails(environmentId).pipe(
```

Comment:
remove the .pipe, close the bracket around the switchMap, and you can reduce indentation by continuing using the parent pipe